app.js

```diff
@@ -26,6 +26,12 @@ app.patch("/api/articles/:article_id", updateVotes);
 
 app.get('/api/users', getAllUsers)
 
+app.get("/api/articles?topic=:topic", getAllArticles)
+
+app.get("/api/articles?sort_by=:property", getAllArticles)
+
+app.get("/api/articles?order=:order", getAllArticles)
+
```

Comment:
queries don't affect the path matching so /api/articles will match with any number of queries. This means you only need one match for app.get("/api/articles")